roles/oVirt.datacenter-cleanup/README.md

```diff
@@ -4,6 +4,16 @@ oVirt Datacenter Cleanup
 The `oVirt.datacenter-cleanup` role is used to cleanup all entities inside
 oVirt datacenters and finally remove the datacenters themselves.
 
+In the special case that the datacenter provided to cleanup contains a VM that
+serves as Hosted Engine the role will not delete the following entities:
+
+- the hosted engine VM itself.
```

Comment:
If you don't start with capital letter here you can omit the trailing dot .  Same bellow in all list

Reply:
will remove the .